rest_api/network_apis/route-route-openshift-io-v1.adoc

```diff
@@ -568,7 +567,6 @@ Defaults to unset
 | Parameter | Type | Description
 | `body`
 | xref:../objects/index.adoc#deleteoptions-meta-v1[`DeleteOptions meta/v1`]
-| 
```

Comment:
@xenolinux Did you intend to remove this row?

Reply:
No. Added new lines